src/index.js

```diff
@@ -0,0 +1,4 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+ReactDOM.render(<p>Hello World!</p>, document.getElementsByClassName('container')[0]);
```

Comment:
please use querySelector or id to retrieve the root element. I know we discourage you to use id-s, but it is a react convention